> The CI failed

```
go: github.com/observatorium/api/api/metrics/v1 imports
        github.com/go-openapi/strfmt imports
        go.mongodb.org/mongo-driver/bson: go.mongodb.org/mongo-driver@v1.17.6: read "https://proxy.golang.org/go.mongodb.org/mongo-driver/@v/v1.17.6.zip": stream error: stream ID 165; INTERNAL_ERROR; received from peer
go: github.com/observatorium/api/api/metrics/v1 imports
        github.com/go-openapi/strfmt imports
        go.mongodb.org/mongo-driver/bson/bsontype: go.mongodb.org/mongo-driver@v1.17.6: read "https://proxy.golang.org/go.mongodb.org/mongo-driver/@v/v1.17.6.zip": stream error: stream ID 165; INTERNAL_ERROR; received from peer
go: github.com/observatorium/api/api/metrics/v1 imports
        github.com/go-openapi/strfmt imports
        go.mongodb.org/mongo-driver/bson/primitive: go.mongodb.org/mongo-driver@v1.17.6: read "https://proxy.golang.org/go.mongodb.org/mongo-driver/@v/v1.17.6.zip": stream error: stream ID 165; INTERNAL_ERROR; received from peer
make: *** [Makefile:98: deps] Error 1
```

Can you restart it?

--- a/api/traces/v1/http_test.go
+++ b/api/traces/v1/http_test.go
@@ -43,7 +43,7 @@ func TestDecompressingTransport(t *testing.T) {
 
 		transport := decompressingTransport(inner)
 
-		req, err := http.NewRequest("GET", "http://example.com", nil)
+		req, err := http.NewRequest("GET", "http://localhost", nil)
 		require.NoError(t, err)
 		req.Header.Set("Accept-Encoding", "br, zstd, gzip")
 
@@ -73,7 +73,7 @@ func TestDecompressingTransport(t *testing.T) {
 
 		transport := decompressingTransport(inner)
 
-		req, err := http.NewRequest("GET", "http://example.com", nil)
+		req, err := http.NewRequest("GET", "http://localhost", nil)
 		require.NoError(t, err)
 
 		resp, err := transport.RoundTrip(req)

